x/orderbook/keeper/participation.go

```diff
@@ -189,6 +189,17 @@ func (k Keeper) CalcWithdrawalAmount(
 		return sdkmath.Int{}, sdkerrors.Wrapf(types.ErrMismatchInDepositorAddress, "%s", bp.ParticipantAddress)
 	}
 
+	bpE, err := k.GetExposureByOrderBookAndParticipationIndex(ctx, marketUID, participationIndex)
+	if err != nil {
```

Comment:
This line can be merged with 196 as follows:
```
if err != nil || len(bpE) == 0 {
...
}
```
also a comment would be nice to understand what are these check for